optimum/exporters/onnx/model_configs.py

```diff
@@ -561,6 +561,13 @@ def inputs(self) -> Dict[str, Dict[int, str]]:
         # TODO: is pixel mask needed?
         return {**super().inputs, "pixel_mask": {0: "batch_size"}}
 
+class DeformableDetrOnnxConfig(ViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 12
```

Comment:
```suggestion
    DEFAULT_ONNX_OPSET = 16
```

otherwise raising `UnsupportedOperatorError: Exporting the operator 'aten::grid_sampler' to ONNX opset version 12 is not supported. Support for this operator was added in 
version 16, try exporting with this version.`

Reply:
Done